### PR TITLE
homepage

### LINKED ISSUE #26: Create home page
Create homepage according to: https://github.com/Abbe-and-Gabbe/jkpgCity/blob/main/JKPGCITY.pdf

### PR DESCRIPTION
**Pull request by saleelala**

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -3,6 +3,14 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <!-- 
+      FONTS
+
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+        integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    -->
```
Comment (albin02forsberg) [inline frontend/index.html]: Remove if not used

File: frontend/src/App.svelte [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <script>
   import Navbar from './Components/Navbar.svelte';
   import Footer from './Components/Footer.svelte';
+  import Homepage from './Venues/Homepage.svelte';
 
 </script>
 
 <Navbar />
+<Homepage />
```
Comment (albin02forsberg) [inline frontend/src/App.svelte]: Merge with main before so you can add routing.

File: frontend/src/Venues/Homepage.svelte [hunk 3]
```diff
@@ -0,0 +1,146 @@
+<!-- @import url('https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&display=swap'); -->
+
+<script>
+    let isCollapsed = true; 
+  
+    function toggleNavbar() {
+      isCollapsed = !isCollapsed;
+    }
```
Comment (albin02forsberg) [inline frontend/src/Venues/Homepage.svelte]: Where is this used?

File: frontend/src/Venues/Homepage.svelte [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+<!--BOD
+<main class="bg-black flex justify-center items-center pt-200 min-h-4 pt-20"> -->
+  
+
+<body class="pt-16">
+
+
+    <!--SECTION 1-->
+    <section class="bg-blue-100 max-w-screen flex flex-wrap items-center justify-between mx-auto p-4 h-auto">
```
Comment (albin02forsberg) [inline frontend/src/Venues/Homepage.svelte]: Sections are too big on "Small" screens.

File: frontend/src/Venues/img/jkpgcity1.jpg
Comment (albin02forsberg) [inline frontend/src/Venues/img/jkpgcity1.jpg:1]: Move this file to 'src/images/**'

**Review by albin02forsberg — CHANGES_REQUESTED** [review]

File: frontend/src/Components/Navbar.svelte [hunk 5]
```diff
@@ -1,38 +1,42 @@
...
-    let isCollapsed = true; 
-  
-    function toggleNavbar() {
-      isCollapsed = !isCollapsed;
-    }
-  </script>
-  
+  import {link} from "svelte-routing"
+
+  let isCollapsed = true;
```
Comment (GabrielAdward) [inline frontend/src/Components/Navbar.svelte:3]: remove line 4-8

File: frontend/src/Homepage/Homepage.svelte [hunk 6]
```diff
@@ -0,0 +1,120 @@
+
+<body class="dark:bg-gray-900 pt-1 pb-3">
+
+
+    <!--SECTION 1-->
```
Comment (GabrielAdward) [inline frontend/src/Homepage/Homepage.svelte]: Remove all <!-->

**Review by GabrielAdward — APPROVED** [review]
LGTM!

**Pull request merged**